### PR TITLE
Update T1562.001.yaml

### PR DESCRIPTION
**Pull request by tccontre**
**Details:**

added several TTP's that we've seen during Awfulshred wiper analysis like:

- clearing history
- disabling history
- clearing cache
- disable swapping of device paging
- reboot system via kernel system request

**Testing:**

tested manually

**Associated Issues:**

https://cert.gov.ua/article/3718487

### CODE REVIEW

**Comment by josehelps** [comment]
Ran the tests, and they all work as expected, awesome work @tccontre 
I added some minor suggestions to improve the documentation.

**Review by josehelps — CHANGES_REQUESTED** [review]

File: atomics/T1562.001/T1562.001.yaml [hunk 1]
```diff
@@ -793,3 +793,68 @@ atomic_tests:
...
+  - linux
+  executor:
+    command: |
+      history -c
+    name: sh
+    elevation_required: true
+- name: Suspend History
+  auto_generated_guid: 94f6a1c9-aae7-46a4-9083-2bb1f5768ec4
+  description: |
+    suspend Shell History seen in Awfulshred wiper- https://unix.stackexchange.com/questions/10922/temporarily-suspend-bash-history-on-a-given-shell
```
[the remark this answers is not in the document]
Comment (tccontre) [reply atomics/T1562.001/T1562.001.yaml:810]: thanks @josehelps noted on that

File: atomics/T1562.001/T1562.001.yaml [hunk 2]
```diff
@@ -793,3 +793,68 @@ atomic_tests:
...
+  executor:
+    command: |
+      echo 1> /proc/sys/kernel/sysrq 
+      echo b> /proc/sysrq-trigger 
+    name: sh
+    elevation_required: true
+- name: Clear Pagging Cache
+  auto_generated_guid: f790927b-ea85-4a16-b7b2-7eb44176a510
+  description: |
+    clear pagging cache via system request. seen in Awfulshred wiper.
```
[the remark this answers is not in the document]
Comment (tccontre) [reply atomics/T1562.001/T1562.001.yaml:835]: @josehelps thanks for this great suggestion and awesome link. I'll add this info

File: atomics/T1562.001/T1562.001.yaml [hunk 3]
```diff
@@ -793,3 +793,68 @@ atomic_tests:
...
+  - linux
+  executor:
+    command: |
+      echo 3> /proc/sys/vm/drop_caches 
+    name: sh
+    elevation_required: true
+- name: Disable Swapping of Paging Device
+  auto_generated_guid: e74e4c63-6fde-4ad2-9ee8-21c3a1733114
+  description: |
+    disable swaaping of device paging. seen in Awfulshred wiper.
```
[the remark this answers is not in the document]
Comment (tccontre) [reply atomics/T1562.001/T1562.001.yaml:846]: noted on this one

**Review by josehelps — APPROVED** [review]

**Comment by josehelps** [comment]
LGTM awesome work @tccontre !

**Pull request merged**